firmware/controllers/algo/rusefi_enums.h

```diff
@@ -59,10 +59,6 @@ typedef enum  __attribute__ ((__packed__)) {
 	 */
 	VVT_MIATA_NB = 3,
 
-	/**
-	 * Single-tooth cam sensor mode where TDC and cam signal happen in the same 360 degree of 720 degree engine cycle
-	 */
-	VVT_FIRST_HALF = 4,
```

Comment:
goodbye `4` :(

should we add a placeholder like `VVT_DEPRECATED` or similar?  or even re-shuffle?